### PR TITLE
More documentation touchups

### PR DESCRIPTION
**Pull request by gordonhart**
Fixes KOL-2499
Fixes KOL-2511
Fixes KOL-2521
Fixes KOL-2524
Fixes KOL-2525
Fixes KOL-2526
Fixes KOL-2513
Fixes KOL-2523

### CODE REVIEW

File: docs/building-a-workflow.md [hunk 1]
```diff
@@ -214,13 +224,13 @@ With basic tests defined for the 300-W dataset, we're almost ready to start test
...
-logic into the [class-based][kolena.workflow.Evaluator] or
-[function-based][kolena.workflow.BasicEvaluatorFunction] evaluator interface.
+logic into the [class-based][kolena.workflow.Evaluator] or [function-based][kolena.workflow.BasicEvaluatorFunction]
+evaluator interface.
 
-Evaluators can have arbitrary [configuration][kolena.workflow.EvaluatorConfiguration], allowing you to evaluate model
-performance under a variety of conditions.
-For this keypoints example, perhaps we want to compute performance at a few different NME threshold values, as this
-threshold drives the `alignment_failure` metric.
+Evaluators can have arbitrary configurationo [(`EvaluatorConfiguration`)][kolena.workflow.EvaluatorConfiguration],
```
Comment (munkyshi) [inline docs/building-a-workflow.md]: sp: configuration

**Comment by munkyshi** [comment]
Feels bad to say, but it now feels like the FAQs are a bit too drab 😅 
Screenshot 2023-06-14 at 23 25 11

**Comment by munkyshi** [comment]
Unrelated to this PR specifically, but the help icon makes me feel like it's the unrecognized unicode character � (especially when repeated the number of times it is on the FAQ page)

**Comment by gordonhart** [comment]
Related to your comments on FAQs @munkyshi — I've added a custom FAQ admonition type with styles matching table styles. Also repurposed the flag icon for FAQs, and changed the "core concepts" icon to something thematically similar to the flag.

Let me know what you think.

**Review by munkyshi — APPROVED** [review]
Looks good!

**Pull request merged**